src/app/grid/grid-editing-excel-style/grid-editing-excel-style.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="grid__wrapper">
+    <igx-grid
+      igxPreventDocumentScroll
+      #grid
+      [data]="data"
+      [primaryKey]="'ProductID'"
+      width="100%"
+      height="500px"
```

Comment:
Please, run both the docfx and the samples and observe that there is much white space between the live editing buttons and the sample itself
![Screenshot 2022-07-05 at 17 34 06](https://user-images.githubusercontent.com/37667452/177352783-7bd6cfea-76d3-4d8d-8d4a-335069af7e96.png)

Reply:
I've adjusted the height values so the sample fits in the designated window, in case it should be bigger, let me know